src/main/java/frc/robot/Robot.java

```diff
@@ -139,6 +142,9 @@ public void teleopInit() {
     if (m_autonomousCommand != null) {
       m_autonomousCommand.cancel();
     }
+    if (DriverStation.getAlliance().get() == Alliance.Red) {
```

Comment:
A comment might be nice here. E.g.:

```suggestion
    // BLUE is the default, so we only need to handle RED here.
    if (DriverStation.getAlliance().get() == Alliance.Red) {
```